### PR TITLE
Add prettier config

### PR DESCRIPTION
**Pull request by mwvd**
> Re urbit/landscape#1403 — "Add prettier for more consistent formatting"

### Notes

Adds [Prettier](https://prettier.io) to our ESLint config to format code (while also not interfering with our ESLint rules).

Also adds a readme to `pkg/npm/eslint-config` that details instructions for installation and usage.

#### Config
- [ ] Test configuration on [urbit/urbit](https://github.com/urbit/urbit) and review output
- [ ] Review rules

#### Initial Implementation
- [ ] Install this configuration in other `pkg` repositories 
- [ ] Add `lint` script to `package.json` in other `pkg` repositories

### CODE REVIEW

File: pkg/npm/eslint-config/index.js [hunk 1]
```diff
@@ -1,204 +1,208 @@
 const env = {
-  "browser": true,
```
Comment (mwvd) [inline pkg/npm/eslint-config/index.js:2]: Imagine it's best to revert the changes to this file before merging this PR, but at least we know it works. 😌

**Comment by zalberico** [comment]
@arthyn @mwvd - I'm going through all issues in urbit/urbit, is this something we want to clean up and merge? If not, I'll close it.

**Comment by arthyn** [comment]
let's just close, we're doing it in new groups and landscape instead
